src/main/java/nova/core/recipes/crafting/ShapedCraftingRecipe.java

```diff
@@ -261,7 +261,7 @@ public void consumeItems(CraftingGrid craftingGrid) {
 			//if (original == consumed)
 			//    continue;
 
-			if (consumed.count() == 0) {
+			if (consumed != null && consumed.count() == 0) {
```

Comment:
https://github.com/NOVA-Team/NOVA-Core/blob/master/src/main/java/nova/core/recipes/crafting/ItemIngredient.java#L98

We should assert that `consumed` is not null here (`Objects.requireNonNull`) and fix it.